js/templates/listingCard.html

```diff
@@ -42,7 +42,7 @@
   </div>
   <% if (ob.vendor) { %>
   <a class="userIconWrapper js-userName">
-    <% const vendorTip = `${ob.vendor.name}${ob.vendor.location ? `  📍  ${ob.vendor.location}` : ''}`;
+    <% const vendorTip = `${ob.vendor.name}${ob.vendor.location ? ` ${ob.vendor.location}` : ''}`;
```

Comment:
I suggested removing location altogether because it won't be clear what that value represents without some type of label.